src/test/ui/issues/issue-30438-c.nll.stderr

```diff
@@ -1,17 +1,19 @@
 error[E0597]: `x` does not live long enough
   --> $DIR/issue-30438-c.rs:19:5
    |
+LL | fn silly<'y, 'z>(_s: &'y Test<'z>) -> &'y <Test<'z> as Trait>::Out where 'z: 'static {
+   |                      ------------     ---------------------------- has type `&'y <Test<'z> as Trait>::Out`
```

Comment:
question: do you think it's good to talk about `'z` here? I'm torn.

(Also, man, it'd be nice if we could just underline `'y` here.)

Reply:
I've changed this so that it only underlines the `'y`. Looking into the types to work out what the `'z` is used for seems like it would be challenging.